src/makeInfinityAdder.js

```diff
@@ -6,7 +6,22 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+  const adder = (num) => {
+    if (num) {
+      sum += num;
+
+      return adder;
+    } else {
```

Comment:
else is redundant

Reply:
At first i didn`t get why, but now i understand